interfaces/automated-testing/scd/scd.yaml

```diff
@@ -36,6 +36,25 @@ components:
           enum: [Starting, Ready]
           example: Ready
 
+    InjectFlightResponse:
+      type: object
+      required:
+        - results
+      properties: 
+        result: 
+          description: >-
+            The result of the flight submission
+
+              - `Planned`: The flight submission data was correct and the operational intent was successfully accepted by the DSS.
+
+              - `Rejected`: The flight submission data was incorrect and / or the USS was not able to successfully submit the operational intent to the DSS.
```

Comment:
I think "Failed" (a problem with the USS or a downstream system) should probably be different from "Rejected" (a problem with the user's submission), so with same style of comments as above, I recommend: `The flight submission data provided was invalid and/or could not be used to attempt to authorize the flight.`

I think adding "Failed" separately is probably useful, so I'd recommend: `The USS was not able to successfully authorize the flight due to a problem with the USS or a downstream system.`

Reply:
Thanks! I have now made changes to this.